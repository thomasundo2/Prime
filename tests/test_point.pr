int main()
{
    pt x;
    printpt([1,2]);
    return 0;
}
