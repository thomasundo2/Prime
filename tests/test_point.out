[1,2]
[99,22]
